src/main/java/com/softserve/repository/SportHubDatabaseRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.softserve.repository;
+
+import java.util.List;
+
+import com.softserve.dto.ArticleDTO;
+
+/**
+ * 
+ * @author José Castellanos
+ * Repository for build sql explicit
+ */
+public interface SportHubDatabaseRepository {
+
+	/**
+	 * 
+	 * @param idCategoria 
+	 * @return List of the Articles ArticleDTO
+	 */
+	List<ArticleDTO> findArticlesFromCategories(Integer idCategoria);
```

Comment:
typo on parameter name "categoryId".

is this method working well? I suppose it should be something like findArticlesByCategoryId(int categoryId)

Reply:
Sure, it's working, we made it because we are implementing DTO objects for not transfer all entities from backend to frontend.